### DISCUSSION TITLE
How to migrate from github lfs storage to rudolfs storage?

### DISCUSSION THREAD

**Discussion by davidvc** [jasonwhite/rudolfs#41] [Q&A] [answered]
I am looking at using rudolfs if standard github lfs becomes too expensive.

I see how you can change the URL for the git-lfs server to point it to the rudolfs server, but it's not clear to me how I get the existing files migrated over to the rudolfs storage (e.g. S3).  Any tips?

Thanks!

**Comment by jasonwhite** [comment, marked as answer]
You can use `git lfs fetch --all` to fetch all LFS objects (that are referenced by commits) from GitHub's LFS storage. Then, once those are downloaded, you can change the URL for the LFS server and push them to the new server with `git lfs push --all`.

**Reply by davidvc** [reply]
Thanks!
